Client/DataAccess/SQL/VisitsTable.swift

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let TableNameVisits = "visits"
+
+class VisitsTable<T>: GenericTable<Visit> {
+    override var name: String { return TableNameVisits }
+    override var rows: String { return "guid TEXT NOT NULL UNIQUE, " +
+                      "siteGuid TEXT NOT NULL, " +
+                      "date REAL NOT NULL" }
```

Comment:
r.e. the second dot here: I'd like to keep some implementation details like the foreign key bits invisible to users. i.e. they'd at best have a url or Site to pass us. I did the other two though, and I can stop using guid's in favor of a numeric ID...